cg/cli/archive.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+import click
+
+from cg.constants.constants import DRY_RUN
+from cg.meta.archive.archive import SpringArchiveAPI
+from cg.models.cg_config import CGConfig
+
+
+@click.group()
+@click.pass_obj
+def archive():
+    """Archive utilities"""
+    pass
+
+
+@archive.command("archive-all-non-archived-files")
```

Comment:
Seems spring centric. Should it be reflected in the cli cmd?

Reply:
Yes, gave it a think, and I think I want it to be `archive-spring-files` with a `--archive-all` option. Feels weird to call it "archive all" if the default behaviour is to archive 200 spring files.

Reply:
Just saw that `archive-spring-files` is the name chosen for the backup API as well - do you think a different name should be chosen here @henrikstranneheim ?

Reply:
Unfortunate, but I think we should name this one without taking the other one in account. The other one should be removed once this in place.